src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -96,6 +96,7 @@ private void initialize() {
 
     public void createButtons() {
 
+        String tester = "Hello World " + "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "tester" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1481)